packages/flutter/lib/src/rendering/object.dart

```diff
@@ -210,9 +202,9 @@ class PaintingContext {
 
   void _startRecording() {
     assert(!_isRecording);
-    _currentLayer = new PictureLayer(canvasBounds);
+    _currentLayer = new PictureLayer(estimatedBounds);
     _recorder = new ui.PictureRecorder();
-    _canvas = new Canvas(_recorder, canvasBounds);
+    _canvas = new Canvas(_recorder, new Rect.largest());
```

Comment:
with https://github.com/flutter/engine/pull/4551 we can just omit the second argument, which is probably simpler.